src/MCPServer/lib/server/packages.py

```diff
@@ -36,6 +37,15 @@
 
 StartingPoint = collections.namedtuple("StartingPoint", "watched_dir chain link")
 
+
+def _get_setting(name):
+    """Retrieve a Django setting decoded as a unicode string."""
+    try:
+        return strToUnicode(getattr(settings, name))
+    except:
```

Comment:
I don't know if I read this entirely correctly, but we could potentially receive a `UnicodeDecodeError` or `AttributeError` here, but for any exception, the function will return `None`. This will cause the joins below to output: `AttributeError: 'NoneType' object has no attribute 'endswith'`? If that's right, is there a reason we don't just let the failure occur here?

Reply:
Ah, good point! I remember thinking of returning `""` instead of `pass` but I think we could just let it fail :+1: